regression_tests.json

```diff
@@ -0,0 +1,7 @@
+{
+    "TestWriteFile": {
+        "difficulty": "basic",
+        "dependencies": [],
+        "test": "agbenchmark/tests/basic_abilities/write_file/write_file_test.py"
+    }
+}
```

Comment:
this is just an example here. the file depends on which agent we run it against.